### ISSUE TITLE
With Docker18.09+nvidia-docker2, cudatoolkit can be installed only inside docker？

### ISSUE THREAD

**Issue by startfromzero0** [NVIDIA/nvidia-docker#1620]
If you use docker18.09+nvidia-docker2, can cudatoolkit be installed only in docker, and the host machine is not installed?

**Comment by elezar** [comment]
Hi @startfromzero0 the CUDA driver must be available on the host, but the CUDA toolkit can be installed in a container. Alternatively one of the `nvidia/cuda` docker images can be used.

**Comment by startfromzero0** [comment]
Thanks for your answer,I still have a question,do I need to install the CUDA driver again in the docker container?

**Comment by elezar** [comment]
No, the driver is only installed on the host. The NVIDIA Container Toolkit ensures that the required driver libraries are made available to the container.
